html/rendering/non-replaced-elements/the-fieldset-and-legend-elements/legend-align-text-align.html

```diff
@@ -0,0 +1,26 @@
+<!doctype html>
+<title>legend align to text-align</title>
```

Comment:
It'd be good to have a more descriptive title, and a link to the spec PR / discussions, maybe.